__tests__/app.test.js

```diff
@@ -120,3 +120,40 @@ describe('GET /api/topics', () => {
     
   });
 
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("200 OK: all comments for an article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeInstanceOf(Array);
+          comments.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id");
```

Comment:
It would be ideal to be a bit more specific and assert that we expect some values for each element as well as just keys.